config/gazebo9_debian_stretch.upstream.yaml

```diff
@@ -0,0 +1,6 @@
+name: gazebo9_import_debian_stretch
+method: http://packages.osrfoundation.org/gazebo/debian-stable
+suites: [stretch]
+component: main
+architectures: [amd64, source]
```

Comment:
We also need arm64 right?

Reply:
I had checked on artful, and there weren't arm64 packages available, but I forgot to double check on stretch.  Good call, I'll add arm64 here.

Reply:
As we don't build debs for arm on artful that's not a big deal. For debian we do need the arm64 ones at least.
Then it would be convenient for users if we this would allow people to build from source, but not a requirement